README.md

```diff
@@ -1 +1,14 @@
-# opp-school-library-app-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Opp-School-library-app
+> Project to create a school database using RUBY
```

Comment:
- [ ] **Required:** Kindly add [more sections](https://github.com/microverseinc/readme-template) like `getting started` , a guide on how to use you application to make it look professional.